### PR TITLE
Add event.dataset to filebeat

### PR DESCRIPTION
**Pull request by ruflin**
This allows 6.6 / 6.7 data to be compatible with 7.x.

### CODE REVIEW

File: x-pack/filebeat/module/suricata/eve/test/eve-alerts.log-expected.json [hunk 1]
```diff
@@ -54,7 +55,7 @@
         "tags": [
             "suricata"
         ], 
-        "url.hostname": "example.net",
+        "url.hostname": "example.net",
```
[the remark this answers is not in the document]
Comment (webmat) [reply x-pack/filebeat/module/suricata/eve/test/eve-alerts.log-expected.json]: Ah I may have saved it in a commit, before I was familiar with the Beats integration tests :-)

File: filebeat/_meta/fields.common.yml [hunk 2]
```diff
@@ -56,6 +56,10 @@
       description: >
         The Filebeat fileset that generated this event.
 
+    - name: event.dataset
+      description: >
+        The Filebeat dataset that generated this event.
+
```
Comment (webmat) [inline filebeat/_meta/fields.common.yml]: Nice, so we have per-Beat override 👍

**Comment by ruflin** [comment]
@graphaelli Good question about the test. I don't think we should enforce it (event though at the moment it is because it's in the code). More important for me is that it's correct in the generate files. Ok if for now we skip a test?

**Comment by graphaelli** [comment]
sounds reasonable to me

**Pull request merged**